### PR TITLE
{math}[foss/2020b]PyStan v2.19.1.1

### PR DESCRIPTION
**Pull request by fizwit**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/p/PyStan/PyStan-2.19.1.1-foss-2020b.eb [hunk 1]
```diff
@@ -0,0 +1,24 @@
+easyblock = 'PythonPackage'
```
Comment (akesandgren) [inline easybuild/easyconfigs/p/PyStan/PyStan-2.19.1.1-foss-2020b.eb:1]: Please make this a PythonBundle. We're trying to move in that direction for all Python packages when possible.

**Review by akesandgren — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/p/PyStan/PyStan-2.19.1.1-foss-2020b.eb (hunk 1 above)
Comment (fizwit) [reply easybuild/easyconfigs/p/PyStan/PyStan-2.19.1.1-foss-2020b.eb:1]: Moving to PythonBundle is good news, will make tools like easy_update operate better.

**Comment by akesandgren** [comment]
@boegelbot Please test @ jsc-zen2

**Review by akesandgren — APPROVED** [review]

**Comment by akesandgren** [comment]
Going in, thanks @fizwit!

**Pull request merged**